Dockerfile

```
@@ -4,7 +4,8 @@ FROM ubuntu:22.04
 ARG DEBIAN_FRONTEND=noninteractive
 
 # tools needed for docker setup
-RUN apt-get update && apt-get install -y apt-utils curl bash
+RUN apt-get update && apt-get install -y apt-utils curl bash && apt-get -y install sudo
+
```

```suggestion
RUN apt-get update && apt-get install -y apt-utils curl bash sudo
```

For simplicity, let's do it in the same `apt-get` command.